src/views/red-experience.njk

```diff
@@ -0,0 +1,126 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+
+{% block content %}
+
+  {% set redExperienceYearsErrors %}
+    {% if model.redExperienceYearsInvalidError %}
+      <p>The number of years experience you have controlling red deer must only contain numbers</p>
+    {% else %}
+      <p>You must enter the number of years experience you have controlling red deer</p>
+    {% endif %}
+  {% endset -%}
+
+  {% set yesHtml %}
+    <h4 class="govuk-heading-s">
+      How many years experience do you have controlling red deer?
+    </h4>
+      {{ govukInput({
+        classes: "govuk-input--width-2",
+        id: "red-experience-years",
+        name: "redExperienceYears",
+        errorMessage: {
+          html: redExperienceYearsErrors
+        } if model.redExperienceYearsInvalidError or model.redEmptyError
+      }) }}
+
+      <h4 class="govuk-heading-s">
+        What is the average number of red deer you have controlled per year over the last 5 years?
+      </h4>
+      {{ govukInput({
+        classes: "govuk-input--width-4",
```

Comment:
This is getting really petty now, but I don't like the width of these controls. 😟

The two digit one looks really out of place to me. And the fact they're different widths seems awkward too.

According to the GDS example, they've used a 10 width input for an 8 digit number, https://design-system.service.gov.uk/components/text-input/#number-input-example, so could we try a 6-width input for both of them and see what it looks like?

Reply:
no worries ill change this to a width of 6